test/models/card_test.rb

```diff
@@ -10,14 +10,25 @@
     expect(card).must_be :valid?
   end
 
-  it "is invalid without a title" do
+  it "is invalid without text or emoji" do
     # Act
-    card.title = nil
+    card.text = nil
+    card.emoji = nil
 
     # Assert
     expect(card.valid?).must_equal false
-    expect(card.errors.keys).must_include :title
-    expect(card.errors[:title]).must_include "can't be blank"
+    expect(card.errors.keys).must_include :text
+    expect(card.errors[:text]).must_include "invalid text or missing emoji"
+  end
+
+  it "is valid with an emoji alone" do
+    card.text = nil
+    VALID_EMOJIS = ["heart_eyes", "beer", "clap", "sparkling_heart", "heart_eyes_cat", "dog"]
```

Comment:
Again this is a place where we can re-use the constant that is defined in the `Board` class.